lib/pactflow/client/provider_contracts/publish.rb

```diff
@@ -26,23 +27,29 @@ def do_call
         end
 
         def render_response(res)
+          notices = [
+            { type: 'success', text: "Successfully published provider contract for #{provider_name} version #{provider_version_number} to Pactflow"},
+          ]
           if res.body && res.body['_links'] && res.body['_links']['pf:ui']['href']
-            ui_url = "\nView the uploaded contract at #{blue(res.body['_links']['pf:ui']['href'])}"
```

Comment:
Have a look at the master branch for the improved version of line 29.

Reply:
I must be blind, looking in the wrong place or it's not there yet

https://github.com/pact-foundation/pact_broker-client/blob/392a872a8e665ecf66eb5cdc18c8414c914f3798/lib/pactflow/client/provider_contracts/publish.rb#LL29C1-L30C100